src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -131,7 +142,7 @@ public void Dispose()
         /// </summary>
         private async Task RunChangeConsumptionLoopAsync()
         {
-            this._logger.LogDebugWithThreadId("Starting change consumption loop.");
+            this._logger.LogInformationWithThreadId($"Starting change consumption loop. BatchSize: {this._batchSize} PollingIntervalMs: {this._pollingIntervalInMs}");
```

Comment:
I think such information should be logged at debug level as user won't be interested in generally in internal workings of the extension. We can log the evaluated batch size and polling interval as separate message.

Reply:
I'm still a little unsure about exactly what should be info and what should be debug, but does this sound reasonable?

info - information useful to the user to see during normal execution
debug - detailed information about the inner workings

The reason I changed this to info was because I was thinking that a user would find it useful to know the batch size and polling interval - especially for the default values which they may not know offhand.

Reply:
I'm going to leave this as info for now for the reasons above, and we can come back and do a look over logging in the entire extension pre-GA to make sure everything lines up how we want (I'll make an issue to track)

Reply:
I already mentioned that we should log batch size and polling interval but not the rest of the message.

Users normally use the logs to check the history of their own function invocations. The extension should in general log all the information that might be useful for the user during host/listener startup only. Logging "Starting change consumption loop" even if for one-time will only confuse users and will prompt them to check what the message means for them.

Reply:
Oh sorry, I misread your original comment. What would you suggest the info message with the batch size/polling interval would be then?